pegasus/sites.v3/code.org/public/learn2016/index.haml

```diff
@@ -104,7 +104,12 @@ social:
 - hide_filters = {}
 - hide_filters[:activity_type] = ["robotics"]
 
--# = tutorialsContents.to_json
+
+- tutorials_contents.each do |tutorial|
+  - tutorial[:contact] = nil
+
+-# = tutorials_contents.to_json
```

Comment:
Oh, I've just left it in forever because it's handy to dump out the list when debugging, but it is a bit unclear so I'll remove it.